modules/setting-up-a-customized-argocd-instance.adoc

```diff
@@ -0,0 +1,80 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/getting-started--with-openshift-gitops.adoc
+
+[id="setting-up-a-customized-argocd-instance_{context}"]
+= Setting up a customized Argo CD instance
+
+The Red Hat OpenShift GitOps operator automatically installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions, allowing Argo CD to manage cluster-scoped resources.
+
+If you want to install your own instance for managing your cluster configurations or deploying your applications, you must deploy your own instance of Argo CD. Using your own instance of Argo CD helps you to continuously deliver your applications.
+
+.Installing ArgoCD
```

Comment:
@jc-berger PTAL at the preview: https://deploy-preview-29411--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/getting-started-with-openshift-gitops.html
these sections are rendered in red and italics. I suggest we change the headings from . to == second level headings everywhere.
Please use th epreview to do a self edit, before you send it out to the OCP team.